cmd/finch/nerdctl_darwin.go

```diff
@@ -46,3 +49,70 @@ func resolveIP(host string, logger flog.Logger, _ command.Creator) (string, erro
 	}
 	return host, nil
 }
+
+// removes the consistency key-value entity from --mount.
+func handleBindMounts(_ NerdctlCommandSystemDeps, _ *config.Finch, nerdctlCmdArgs []string, index int) error {
```

Comment:
do we need to pass args that are not getting used or they needed for different architectures?

Reply:
The handler interface is set by [nerdctl.go line46](https://github.com/runfinch/finch/pull/1100/files#diff-403cda46333c3f741f8b1ff7f391901b0ccb8a5501cdd241138f89fd3c9db2c6R46), so to be "least impactful" these new handlers conformed to the same signature, but did not necessarily require the same args.